README.md

```diff
@@ -92,6 +92,7 @@ Click the button below to start deployment.
    - **Event Hub Connection String** - Type the connection string for the existing Event Hub; leave empty if creating a new Event Hub.
    - **Event Hub Namespace** - Type the namespace for the existing Event Hub; leave empty if creating a new Event Hub.
    - **Event Hub Name** - Type the name of the existing Event Hub.
+   - **Event Hub Filter** - To check what type of filtering is supported click [here](#event-hub-filtering)
```

Comment:
according to the [template](https://github.com/alertlogic/ehub-collector/blob/v1/templates/ehub.json#L64-L77) there are two filter parameters: `Event Hub Filter Json` and `Event Hub Filter Regex`. PLease put them both here.

Reply:
Done